Team-#19 README.md 단장하기

ksundong opened the pull request:
Fixes #19

## 설명

README.md 파일을 업데이트 했어요!

**추가한 항목**  
-  멤버 관련 정보
-  Wiki 링크
-  API 문서 링크
-  젠킨스는 뱃지 안되나? 뱃지넣기
-  tasklist 링크
-  참조해서 예쁘게 만들기
-  프로젝트 빌드방법 작성

## 작업 유형

- [x] 관련 문서 업데이트 필요

## 체크리스트

- [x] 변경사항에 대한 문서를 작성하였는가?

ksundong committed:
Team-#19 docs: 멤버 관련 정보 추가

ksundong committed:
Team-#19 docs: 뱃지 추가, 링크 추가

ksundong committed:
Team-#19 docs: 빌드 방법 정리

ksundong committed:
Team-#19 docs: 프로젝트 관리 내용 작성

ksundong committed:
Team-#19 fix: 이미지 깨지는 이슈 수정

ksundong committed:
Team-#19 docs: 뱃지들이 사회적 거리두기를 준수하지 않아 수정

reesekimm commented on README.md:
````diff
@@ -1,2 +1,154 @@
+- [ ] 새로운 테스트와 기존의 테스트가 변경사항에 대해 만족하는가?
+```
+
+### 우리는 어떻게 커뮤니케이션 하는가?
````
이런 섹션 좋네요ㅎㅎ👍

ksundong replied:
3조꺼 좀 참고했습니다 ㅋㅋㅋ

haveagood commented:
<img width="671" alt="스크린샷 2020-06-11 오전 11 33 58" src="https://user-images.githubusercontent.com/42365037/84338598-73f55e00-abd7-11ea-9601-e4df1bb51262.png">

이부분 진짜 잘 만드신거 같습니다.
뭔가 이번 팀에서는 새로운 기능 새로운 방법을 엄청 많이 보는거 같네요 ㅋㅋ 내가 여태까지 너무 대충했나...

reesekimm commented on README.md:
```diff
@@ -1,2 +1,154 @@
+
+### 개발 환경 관리(Environment Management)
+
+- 각 Class 별 별도 디렉토리에 소스코드를 위치시킨다.(BE/FE) Sub-Module은 사용하지 않는다.
```
(질문) Sub-Module을 사용하지 않는다는게 뭔가요?

ksundong replied:
Git의 sub-module이라고, .git으로 버전관리되는 디렉토리 내부에 .git으로 관리되는 디렉토리가 있다면 sub-module이 됩니다.
이 경우에는 관리가 복잡해져서 사용하지 않는다고 적었어요!
[참고](https://git-scm.com/book/ko/v2/Git-%EB%8F%84%EA%B5%AC-%EC%84%9C%EB%B8%8C%EB%AA%A8%EB%93%88)

reesekimm replied:
@ksundong 
git 서브모듈.. 유용한 도구이지만 잘 다루려면 엄청난 내공이 필요해 보이네요..
덕분에 git 서브모듈의 존재에 대해 알게됐네요ㅎㅎ
고맙습니다:)

reesekimm commented on README.md:
```diff
@@ -1,2 +1,154 @@
+
+or background deploy `nohup java -jar build/lib/1.0.0-Octocat.jar &`
+
+#### FE
```
이 부분 추후 업데이트 하겠습니다😊

ksundong replied:
네~

haveagood reviewed (approved):
고생하셨습니다!!!

reesekimm commented on README.md:
```diff
@@ -1,2 +1,154 @@
+  - 각 주차별로 Phase-{weeknum}을 붙여서 Milestone을 정의한다.
+  - 주의 시작에 간단하게 Milestone을 정의한다.
+
+- BE, Team의 경우 Github의 Project Kanban을 활용, 완료된 항목과 진행중인 항목이 눈에 띌 수 있도록 한다.
```
FE 추가해주세요!

ksundong replied:
앗 넵!

reesekimm reviewed (approved):
고생하셨습니다!
issue/pr template 작성이나 README 작성 같이 Team 관련 작업을 주도적으로 진행해주셔서 항상 고맙습니다😀👍

ksundong committed:
Team-#19 docs: 리뷰내용 반영(칸반에 FE도 추가)

참조(https://github.com/codesquad-member-2020/issue-tracker-01/pull/27#discussion_r438510727)
